.github/workflows/build.yaml

```diff
@@ -319,28 +319,28 @@ jobs:
 
       - uses: ./.github/actions/handle-tagged-build
 
-      - uses: actions/download-artifact@v4
+      - uses: ./.github/actions/download-artifact-retry
```

Comment:
[not about this line] When I search for `actions/download-artifact@v4` in the repo, I get hits in other files such as `release-ci.yaml`, `scanner-db-init-dump.yaml` and `scanner-dev-vuln-update.yaml`. Do you mind updating them also?